vortex-array/src/compute/mod.rs

```diff
@@ -25,7 +27,7 @@ pub use take::{take, TakeFn};
 
 use crate::ArrayData;
 
-mod binary_numeric;
+pub mod binary_numeric;
```

Comment:
Let's not make this pub

Reply:
Done. I added a `pub mod test_harness` to this file and `pub use`d `test_binary_numeric` in there instead of conditionally `pub use`ing it in the regular module.